lib/rails_admin_nestable/nestable.rb

```diff
@@ -23,7 +23,7 @@ class Nestable < Base
         register_instance_option :controller do
           Proc.new do |klass|
             @nestable_conf = ::RailsAdminNestable::Configuration.new @abstract_model
-            @position_field = @nestable_conf.options[:position_field]
+            @position_field = @nestable_conf.options[:position_field].to_s.split('.').last
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.